### ISSUE TITLE
Missing 01/17's inductor-A100-perf-nightly results in HUD

### ISSUE THREAD

**Issue by xmfan** [pytorch/pytorch#118012]
### 🐛 Describe the bug

We are missing an entry for 01/17 in the dashboard, but all jobs have completed successfully. The missing run's result is https://github.com/pytorch/pytorch/actions/runs/7552164821.

<img width="325" alt="image" src="https://github.com/pytorch/pytorch/assets/9547562/42056371-0161-4e2b-b980-217636d88be4">

### Versions

5667a990fd0aaa1cec2df8f6666fae9df478ee03

cc @seemethere @malfet @pytorch/pytorch-dev-infra @ezyang @msaroufim @bdhirsh @anijain2305 @zou3519

**Comment by huydhn** [comment]
I guess GitHub asw misbehaving again and failed to upload the results, let me take a look

**Comment by huydhn** [comment]
jan 17th data is available now https://hud.pytorch.org/benchmark/compilers?startTime=Tue%2C%2016%20Jan%202024%2000%3A23%3A03%20GMT&stopTime=Tue%2C%2023%20Jan%202024%2000%3A23%3A03%20GMT&granularity=hour&suite=torchbench&mode=training&dtype=amp&lBranch=main&lCommit=8f7caaee67b1799ac84a99ac508aea630b28f124&rBranch=main&rCommit=5667a990fd0aaa1cec2df8f6666fae9df478ee03

TLDR; I need to rerun the job manually https://github.com/pytorch/pytorch/actions/runs/7563673741/job/20747672983 with a small tweak to set its date to Jan 17th instead of today.

Let me know if it works.

**Comment by xmfan** [comment]
Looks good, thanks Huy!
